Gemfile.lock

```diff
@@ -403,7 +392,7 @@ DEPENDENCIES
   webmock
 
 RUBY VERSION
-   ruby 2.5.5p157
+   ruby 2.5.8p224
 
 BUNDLED WITH
-   2.0.2
+   2.1.4
```

Comment:
Curious – did you modify `Gemfile.lock` directly? Also wondering if there's anything we will want to look out for by updating bundler.

Interesting side note: since we only specify `2.5` on the container, `ruby -v` on the container spits out: `ruby 2.5.8p224`. It must defer to patches in the same way that `setup-ruby` does

Reply:
Nope, I just did `rbenv install 2.5.8 && gem install bundler && bundle install`